.eslintrc.js

```diff
@@ -17,7 +17,11 @@ module.exports = {
 
   overrides: [
     {
-      files: ['**/__tests__/**/*.js', '**/__mocks__/**/*.js'],
+      files: [
+        '**/__tests__/**/*.js',
+        '**/__mocks__/**/*.js',
+        'acceptance-tests/tests/**/*.js',
```

Comment:
Since these overrides are for jest, is this what we want?

Reply:
I moved this into a specific override for jasmine